### ISSUE TITLE
A possibility to update dataset processing for TUH v1.5.4

### ISSUE THREAD

**Issue by MohSamNaf** [Eldave93/Seizure-Detection-Tutorials#4]
Hello David,
Thank you so much for providing these useful notebooks.
I would like to ask if there is a possibility that you might update the [01. Overview of Datasets.ipynb] part for TUH Dataset in the near future.
NEDC has came up with a new file structure for seizure labeling starting from v1.5.3. 

Thank you for your great work!!!

**Comment by jsaleeb** [comment]
Agreed, would be great if that can be updated for v2.0.3 to read the appropriate files.
